src/User32.Desktop/User32.cs

```diff
@@ -621,10 +674,478 @@ public static partial class User32
+        /// <param name="lParam">An application-defined value to be passed to the callback function.</param>
+        /// <returns>
+        /// If the function is unable to perform the enumeration, the return value is zero. Call GetLastError to get extended error information.
+        /// If the function succeeds, it returns the nonzero value returned by the callback function that was pointed to by <paramref name="lpEnumFunc"/>.
+        /// If the callback function fails, the return value is zero. The callback function can call SetLastError to set an error code for the caller to retrieve by calling GetLastError.
+        /// </returns>
+        /// <remarks>
+        /// The EnumWindowStations function enumerates only those window stations for which the calling process has the <see cref="WINDOW_STATION_ACCESS_MASK.SpecificRight.WINSTA_ENUMERATE"/> access right.
+        ///  EnumWindowStations repeatedly invokes the <paramref name="lpEnumFunc"/> callback function until the last window station is enumerated or the callback function returns FALSE.
+        /// </remarks>
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool EnumWindowStations(EnumWindowStationDelegate lpEnumFunc, IntPtr lParam);
+
+        /// <summary>
+        /// Enumerates all window stations in the current session. The function passes the name of each window station, in turn, to an application-defined callback function.
+        /// </summary>
+        /// <param name="lpEnumFunc">An application-defined <see cref="EnumWindowStationDelegate"/> callback function.</param>
+        /// <param name="lParam">An application-defined value to be passed to the callback function.</param>
+        /// <returns>
+        /// If the function is unable to perform the enumeration, the return value is zero. Call GetLastError to get extended error information.
+        /// If the function succeeds, it returns the nonzero value returned by the callback function that was pointed to by <paramref name="lpEnumFunc"/>.
+        /// If the callback function fails, the return value is zero. The callback function can call SetLastError to set an error code for the caller to retrieve by calling GetLastError.
+        /// </returns>
+        /// <remarks>
+        /// The EnumWindowStations function enumerates only those window stations for which the calling process has the <see cref="WINDOW_STATION_ACCESS_MASK.SpecificRight.WINSTA_ENUMERATE"/> access right.
+        ///  EnumWindowStations repeatedly invokes the <paramref name="lpEnumFunc"/> callback function until the last window station is enumerated or the callback function returns FALSE.
+        /// </remarks>
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool EnumWindowStations(IntPtr lpEnumFunc, IntPtr lParam);
```

Comment:
It would be nice to have some Friendly helpers but it's not so often that a pointer / IntPtr version is needed. `SetWindowsHookEx` is special as if you use it on another process (Not your own) you can't pass a managed delegate there, there is now way to make it work (You need to get the pointer from LoadLibrary+GetProcAddress and as it's not in your address space, can't convert it to a managed delegate)

Most functions don't have this limitation as you can't use them to execute code in another process address space :)